packages/rancher-sriov/generated-changes/patch/templates/operator.yaml.patch

```diff
@@ -0,0 +1,70 @@
+--- charts-original/templates/operator.yaml
++++ charts/templates/operator.yaml
+@@ -1,7 +1,22 @@
++{{- if not (.Capabilities.APIVersions.Has "k8s.cni.cncf.io/v1/NetworkAttachmentDefinition") -}}
++{{- required "rke2-multus is required but not found!" "" -}}
++{{- end -}}
++apiVersion: sriovnetwork.openshift.io/v1
++kind: SriovOperatorConfig
++metadata:
++  name: default
++  namespace: {{ .Release.Namespace }}
++spec:
++  # Add fields here
++  enableInjector: {{ .Values.operator.enableAdmissionController }}
++  enableOperatorWebhook: {{ .Values.operator.enableAdmissionController }}
++  configDaemonNodeSelector: {feature.node.kubernetes.io/network-sriov.capable: "true"}
++---
+ apiVersion: apps/v1
+ kind: Deployment
+ metadata:
+   name: {{ include "sriov-network-operator.fullname" . }}
++  namespace: {{ .Release.Namespace }}
+   labels:
+   {{- include "sriov-network-operator.labels" . | nindent 4 }}
+ spec:
+@@ -25,7 +40,7 @@
+       serviceAccountName: {{ include "sriov-network-operator.fullname" . }}
+       containers:
+         - name: {{ include "sriov-network-operator.fullname" . }}
+-          image: {{ .Values.images.operator }}
++          image: {{ include "system_default_registry" . }}{{ .Values.images.operator }}
```

Comment:
It's ugly but it's in the value already:
```
images:
  operator: rancher/hardened-sriov-network-operator:v1.0.0-build20210429
 ```

Reply:
I think we should patch this chart to have the values.yaml be more consistent with other charts, w.r.t. the image tag, such as:

```
images:
  operator:
    image: rancher/hardened-sriov-network-operator
    tag: v1.0.0-build20210429
    os: linux # optional, but if we eventually support windows this will be necessary for airgap too
```

We can then patch all references here accordingly, e.g. 
```suggestion
+          image: {{ include "system_default_registry" . }}{{ .Values.images.operator.image }}{{ .Values.images.operator.tag }}
```